source/server/BUILD

```diff
@@ -258,8 +258,26 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "listener_manager_lib",
```

Comment:
name this listener_lib, and the one below listener_manager_lib?

Reply:
Done. Along with other clean up.

Reply:
Done. Along with other clean up